src/App.js

```diff
@@ -1,11 +1,36 @@
-import React from 'react';
-
+import React from "react";
+import CoinList from './pages/CoinList/CoinList';
+import CoinPage from './pages/CoinPage/CoinPage';
+import Portfolio from './pages/Portfolio/Portfolio';
+import Navbar from './components/Navbar/Navbar';
+import {Switch, Route} from 'react-router-dom';
```

Comment:
node modules to be imported before the ones you created
use the file structure and setup absolute imports